app/src/androidTest/java/com/github/fribourgsdp/radio/activities/WorkingJoinGameActivityTest.kt

```diff
@@ -93,6 +99,23 @@ class WorkingJoinGameActivityTest {
     }
 
 
+    @Test
+    fun pressBackWorks(){
+        val context: Context = ApplicationProvider.getApplicationContext()
+
+        val intent = Intent(context, VoiceOverIPActivity::class.java)
+        ActivityScenario.launch<VoiceOverIPActivity>(intent).use { _ ->
```

Comment:
Why are you launching `VoiceOverIPActivity` in the tests for the `JoinActivity`?

Reply:
well seen

Reply:
If that is the case, also use the rule so that the same activity is not launched twice.